site/src/sections/Navbar/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { NavStyleWrapper } from "./navbar.style";
+import pattern from "../../assets/images/brand/SVG/service-mesh-patterns-side.svg";
+import Layer5 from "../../assets/images/brand/SVG/Layer5.svg";
+
+const Navbar = () => {
+  return (
+    <NavStyleWrapper>
+      <nav>
+        <ul className="navbar">
+          <img className="pattern" src={pattern} alt="pattern" />
+          <li>Book Chapters</li>
+          <li>Patterns In Action</li>
+          <li>Meet the Authors</li>
+          <li>About <img className="pattern" src={Layer5} alt="Layer5" /> </li>
```

Comment:
Layer5 image should be aligned with the text above (i.e. About)

Reply:
aligned it :+1: